slides/01_intro.qmd

```diff
@@ -26,36 +26,46 @@ image: thumbnails/hello.jpg
 :::
 :::
 
-## Friedrich
+## Liming
 
 ::: columns
 ::: {.column width="70%"}
--   Since 2008 self-employed consultant for computer science and data science as well as biostatistics
--   Co-founder and CEO of RPACT, a company developing the formally validated R package [rpact](https://cran.r-project.org/package=rpact "rpact: Confirmatory Adaptive Clinical Trial Design and Analysis") with 21 releases on CRAN since 2018
--   Trained software architect; R programmer since 2004; R Shiny developer since 2019
+-   Master in Biostatistics from Fudan University
+-   Statistical Programmer in Roche for 4 years, technical engineering lead for chevron team in NEST project
+-   Member of ASA BIOP working group on Software Engineering (SWE WG) mmrm taskforce
+-   Feel free to connect at [`r fontawesome::fa("github")` Github](https://github.com/clarkliming "Liming GitHub")
+:::
+
+::: {.column width="30%"}
+![](resources/liming.jpg)
+:::
+:::
+
+## Joe
+
+::: columns
+::: {.column width="70%"}
+-   ...
 -   Feel free to connect at [`r fontawesome::fa("linkedin")` LinkedIn](http://www.linkedin.com/in/pahlke "Friedrich Pahlke at LinkedIn") or [`r fontawesome::fa("github")` Github](https://github.com/fpahlke "Friedrich Pahlke at GitHub")
 :::
 
 ::: {.column width="30%"}
-![](resources/friedrich_pahlke.jpg)
+<!-- ![](resources/friedrich_pahlke.jpg) -->
 :::
 :::
 
-## Kevin
+## Shuang
 
 ::: columns
 ::: {.column width="30%"}
-![](resources/kevin.jpg)
+![](resources/Shuang.jpg)
 :::
 
 ::: {.column width="70%"}
--   Trained in math, 'statistical engineer' by heart
--   Research associate in Heidelberg & Cambridge
-    -   optimal clinical trial design
-    -   statistical genetics & high dimensional data
--   Methodology statistician with BI since '21
--   R-tist, extensive experience with R packages, version control, and high-performance computing
--   feel free to connect at `r fontawesome::fa("linkedin")` [LinkedIn](https://www.linkedin.com/in/kevin-kunzmann-6486a11bb/) or `r fontawesome::fa("github")` [Github](https://github.com/kkmann)
+-   Master in Pharmacology from Fudan University
+-   5-year experience in clinical data management, curation and related R tools development
+-   Technical Lead of several data FARIification R packages, co-auther of the Restrict Data Check Shiny App and the Raw Data Compare R Package
+-   Feel free to connect at [`r fontawesome::fa("github")` Github](https://github.com/shuangli22 "Shuang GitHub")
 :::
```

Comment:
@Stevemun if you like to have a slide please add one - if you prefer staying in background also ok of course, up to you